docs/synapse/userguides/ug051_ing_tool.rst

```diff
@@ -0,0 +1,67 @@
+Ingest - Commandline Ingest Tool
+===============================
+
+While the Ingest subsystem in Synapse lives at synapse.lib.ingest, most users may use the standalone ingest tool
+directly.  This can be invoked with the following command: ``python -m synapse.tools.ingest <options> <ingest files>``.
+If invoked with the --help flag, the following options are listed::
+
+    ~/synapse$ python -m synapse.tools.ingest --help
```

Comment:
This is really an environmental/distribution dependent thing.  Even in light of PEP394, we have no idea what peoples python's actually are.